instance.cc

```diff
@@ -2908,6 +2965,9 @@ Heightfield PGInstance::getHeightField(float bx, float bz) {
     // acc height
     std::unordered_map<unsigned char, float> biomeCounts(numBiomes);
     float totalHeightFactors = 0;
+    // acc material
+    std::unordered_map<uint8_t, MaterialCountWeightPair> materialsCountWeightMap(numMaterials);
```

Comment:
Why a map? If the number of biomes is known, then this could be a `vector`, right?

Reply:
Yeah that makes sense, 
I can say the same thing for biomes + materials, so I changed both to be a vector